src/Components/Header/Header.js

```diff
@@ -0,0 +1,14 @@
+import './Header.css';
+import puzzle from '../../assets/oh-my-word-puzzle.png';
```

Comment:
Oooo, did you make a logo for your puzzle?  Niceeeee

Reply:
Yes, it's super cute ☺️ in my humble opinion.